crates/ethereum-rpc/src/gas_price/gas_oracle.rs

```diff
@@ -363,6 +363,7 @@ impl<C: sov_modules_api::Context> GasPriceOracle<C> {
                 // check if coinbase
                 let sender = tx.from;
                 sender != block.header.miner
+                    && sender != address!("deaddeaddeaddeaddeaddeaddeaddeaddeaddead")
```

Comment:
Suggestion here:
- Make this address a constant value that we can import and use rather than hard code.

Reply:
https://github.com/chainwayxyz/citrea/issues/675